src/hooks/check-id.js

```diff
@@ -0,0 +1,15 @@
+// Use this hook to manipulate incoming or outgoing data.
+// For more information on hooks see: http://docs.feathersjs.com/api/hooks.html
+const { Forbidden } = require('@feathersjs/errors')
+
+// eslint-disable-next-line no-unused-vars
+module.exports = (options = {}) => {
```

Comment:
Is there a reason to disable no-unused-vars and have the options param? we don't seem to be using it

Reply:
That comes in the hook when it is generated by the cli. I didn't want to get rid of things until I understand them better. I'm not sure exactly how they are used and haven't taken the time to look into it yet.

Reply:
I figured it was a long those lines, thanks